src/measure_theory/measure_space.lean

```diff
@@ -313,6 +313,16 @@ begin
   rw [← measure_union disjoint_diff h₂ (h₁.diff h₂), union_diff_cancel h]
 end
 
+lemma inter_add_inter_compl {α:Type*} [measurable_space α]
+  (μ : measure α) (s t : set α)  (A2 : is_measurable s) (A1 : is_measurable t) :
+  (μ (s ∩ t) + μ (s ∩ tᶜ)) = μ s :=
```

Comment:
This lemma is almost the same as `measure_eq_inter_diff`. Could you use that in your application instead?